src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -1,47 +1,238 @@
 package core.basesyntax;
 
 import java.util.List;
+import java.util.Objects;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private static final String INDEX_OUT_OF_BOUNDS_MESSAGE =
+            "Index: is out of bound for length ";
+    private Node<T> head;
+    private Node<T> tail;
+    private int size = 0;
```

Comment:
int type default value is 0, no need to initialize it here

Reply:
agree fixed